frontend/js/src/metadata-viewer/MetadataViewer.tsx

```diff
@@ -286,7 +310,13 @@ export default function MetadataViewer(props: MetadataViewerProps) {
             aria-labelledby="headingOne"
           >
             <div className="panel-body">
-              <TagsComponent tags={metadata?.tag?.recording} />
+              {Boolean(metadata?.recording) && (
```

Comment:
I am not sure what's the purpose of this check. Is it to check if we have recording mbid to vote on? In that case, it would make more sense to do `Boolean(recordingMBID)` instead.